packages/noports_core/lib/src/sshrvd/build_env.dart

```diff
@@ -1,4 +1,4 @@
 class BuildEnv {
-  static final bool enableSnoop =
-      bool.fromEnvironment('ENABLE_SNOOP', defaultValue: false);
+  static final bool enableSnoop = true;
+      // bool.fromEnvironment('ENABLE_SNOOP', defaultValue: false);
```

Comment:
Are we eliminating the debug build?

Reply:
this was for my convenience while actively developing this PR's features